go.mod

```diff
@@ -0,0 +1,8 @@
+module anki
+
+go 1.13
```

Comment:
Thanks for doing this. It would be good to set this to `go 1.11`, though, since there are no 1.13-specific features used in this module.